src/test/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/javadocTags/expectedAsteriskInJavadocInlineTagAst.txt

```diff
@@ -2,10 +2,10 @@ JAVADOC -> JAVADOC [0:0]
 |--JAVADOC_INLINE_TAG -> JAVADOC_INLINE_TAG [0:0]
 |   |--JAVADOC_INLINE_TAG_START -> { [0:0]
 |   |--LINK_LITERAL -> @link [0:1]
-|   |--NEWLINE -> \r\n [0:6]
```

Comment:
I'm not sure whether this change is suitable in current PR. @romani @rnveach please share your opinion